.vib/wildfly/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/wildfly"
         },
-        "runtime_parameters": "d2lsZGZseVVzZXJuYW1lOiAidXNlciIKd2lsZGZseVBhc3N3b3JkOiAiQ29tcGxpY2F0ZWRQYXNzd29yZDEyMyE0IgpleHBvc2VNYW5hZ2VtZW50Q29uc29sZTogdHJ1ZQppbml0Q29udGFpbmVyczoKIC0gbmFtZTogZG93bmxvYWQtd2FyCiAgIGltYWdlOiBiaXRuYW1pL2JpdG5hbWktc2hlbGwKICAgY29tbWFuZDogWydiYXNoJywgJy1jJywgJ2N1cmwgLWsgaHR0cHM6Ly90b21jYXQuYXBhY2hlLm9yZy90b21jYXQtMTAuMC1kb2MvYXBwZGV2L3NhbXBsZS9zYW1wbGUud2FyIC1vIC9iaXRuYW1pL3dpbGRmbHkvc2FtcGxlLndhciddCiAgIHZvbHVtZU1vdW50czoKICAgICAtIG5hbWU6IGRhdGEKICAgICAgIG1vdW50UGF0aDogL2JpdG5hbWkvd2lsZGZseQpzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cDogODAKICAgIG1nbXQ6IDk5OTAK",
+        "runtime_parameters": "d2lsZGZseVVzZXJuYW1lOiAidXNlciIKd2lsZGZseVBhc3N3b3JkOiAiQ29tcGxpY2F0ZWRQYXNzd29yZDEyMyE0IgpleHBvc2VNYW5hZ2VtZW50Q29uc29sZTogdHJ1ZQppbml0Q29udGFpbmVyczoKIC0gbmFtZTogYnVpbGQtd2FyCiAgIGltYWdlOiBiaXRuYW1pL2phdmE6MTEKICAgc2VjdXJpdHlDb250ZXh0OgogICAgIHJ1bkFzVXNlcjogMAogICBlbnY6CiAgICAgLSBuYW1lOiBNQVZFTl9WRVJTSU9OCiAgICAgICB2YWx1ZTogMy44LjYKICAgICAtIG5hbWU6IEFQUF9WRVJTSU9OCiAgICAgICB2YWx1ZTogInt7IC5DaGFydC5BcHBWZXJzaW9uIH19IgogICBjb21tYW5kOiBbJ2Jhc2gnLCAnLWMnLCAiY2QgL29wdC9iaXRuYW1pICYmIFwKICAgICBpbnN0YWxsX3BhY2thZ2VzIGN1cmwgY2EtY2VydGlmaWNhdGVzICYmIFwKICAgICBjdXJsIC1MIFwiaHR0cHM6Ly9kbGNkbi5hcGFjaGUub3JnL21hdmVuL21hdmVuLTMvJHtNQVZFTl9WRVJTSU9OfS9iaW5hcmllcy9hcGFjaGUtbWF2ZW4tJHtNQVZFTl9WRVJTSU9OfS1iaW4udGFyLmd6XCIgLW8gL29wdC9iaXRuYW1pL21hdmVuLnRhci5neiAmJiBcCiAgICAgdGFyIC14emYgL29wdC9iaXRuYW1pL21hdmVuLnRhci5neiAmJiBcCiAgICAgZXhwb3J0IFBBVEg9XCIvb3B0L2JpdG5hbWkvYXBhY2hlLW1hdmVuLSR7TUFWRU5fVkVSU0lPTn0vYmluOiRQQVRIXCIgJiYgXAogICAgIGN1cmwgLUwgXCJodHRwczovL2dpdGh1Yi5jb20vd2lsZGZseS9xdWlja3N0YXJ0L2FyY2hpdmUvcmVmcy90YWdzLyR7QVBQX1ZFUlNJT059LkZpbmFsLnRhci5nelwiIC1vIC9vcHQvYml0bmFtaS93aWxkZmx5LXF1aWNrc3RhcnQudGFyLmd6ICYmIFwKICAgICB0YXIgLXh6ZiAvb3B0L2JpdG5hbWkvd2lsZGZseS1xdWlja3N0YXJ0LnRhci5neiAmJiBcCiAgICAgY2QgL29wdC9iaXRuYW1pL3F1aWNrc3RhcnQtJHtBUFBfVkVSU0lPTn0uRmluYWwvaGVsbG93b3JsZCAmJiBcCiAgICAgbXZuIGNsZWFuIHBhY2thZ2UgJiYgXAogICAgIGNwIC9vcHQvYml0bmFtaS9xdWlja3N0YXJ0LSR7QVBQX1ZFUlNJT059LkZpbmFsL2hlbGxvd29ybGQvdGFyZ2V0L2hlbGxvd29ybGQud2FyIC9iaXRuYW1pL3dpbGRmbHkvaGVsbG93b3JsZC53YXIiXQogICB2b2x1bWVNb3VudHM6CiAgICAgLSBuYW1lOiBkYXRhCiAgICAgICBtb3VudFBhdGg6IC9iaXRuYW1pL3dpbGRmbHkKc2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBwb3J0czoKICAgIGh0dHA6IDgwCiAgICBtZ210OiA5OTkwCg==",
```

Comment:
I agree with the new approach given that the war now comes directly from Wildfly and we are following their doc to build and deploy the sample app. Just a minor thing, could you add a small comment explaining what the initContainer does and why? This could be confusing to others in the future.

Reply:
Sure, I updated the runtime_parameters with the next comment

```
 # This initContainer builds the sample helloworld.war artifact
 # from the official wildfly/quickstart repository. This is
 # then mounted into de WildFly container to verify the
 # server can deploy a war file. 
```